### DISCUSSION TITLE
Shock tube problem in SU2 ver 7.3.1

### DISCUSSION THREAD

**Discussion by PratipGhoshal** [su2code/SU2#1734] [Q&A]
Hello, I am a new user of SU2. Is it possible to replicate a shock tube problem in SU2 in version 7.3.1. The issue that I am facing is dividing the geometry into driver and driven sections and assigning different pressure and temperatures for different sections. I will be thankful for your suggestions regarding this.

**Comment by WallyMaier** [comment]
Hi PratipGhostal,

I have done some similar problems.   The two things I did were: 1.) Run the solution to steady using the preshock conditions.  Then restart the solution using the a supersonic inlet for the shocked state.

2.) Manually alter a restart file to have your desired states.
